src/core/include/openvino/op/convolution.hpp

```diff
@@ -127,28 +127,6 @@ class OPENVINO_API ConvolutionBackpropData : public util::ConvolutionBackPropBas
     /// \return The output spatial dimensions shape.
     const PartialShape get_output_shape() const;
     void set_output_shape(const Shape& output_shape);
-
-    /// \brief      Calculates output spatial features size.
-    ///
-    /// \param[in]  input_data_shape      The input data partial shape
-    /// \param[in]  filters_shape         The filters partial shape
-    /// \param[in]  strides               The strides values.
-    /// \param[in]  dilations             The dilations values.
-    /// \param[in]  pads_begin            The paddings at the beginning of axis.
-    /// \param[in]  pads_end              The paddings at the end of axis.
-    /// \param[in]  output_padding    The output padding values.
-    /// \param      output_spatial_shape  The placeholder for computed output spatial partial
-    /// shape.
-    ///
-    OPENVINO_DEPRECATED("This member function is deprecated and will be removed soon.")
-    void infer_conv_backprop_output_spatial_shape(const std::vector<Dimension>& input_data_shape,
-                                                  const std::vector<Dimension>& filters_shape,
-                                                  const Strides& strides,
-                                                  const Strides& dilations,
-                                                  const CoordinateDiff& pads_begin,
-                                                  const CoordinateDiff& pads_end,
-                                                  const CoordinateDiff& output_padding,
-                                                  std::vector<Dimension>& output_spatial_shape);
```

Comment:
@ilya-lavrenov @vurusovs This API is used in VPUX Plugin, any replacement for this API?
https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/blob/d9a5ea6e1d4888bba55d37e4f8efd9d38365aed6/src/vpux_compiler/src/dialect/IE/ops/convolution_backprop_data.cpp#L97-L105

Reply:
@praasz @t-jankowski could you take a look?

Reply:
Create e.g `GroupConvolutionBackpropData` operator and take its output shape.